code/modules/mob/living/death.dm

```diff
@@ -27,20 +27,24 @@
 	return
 
 /mob/living/dust(just_ash, drop_items, force)
-	death(TRUE)
+	Stun(100, ignore_canstun=TRUE) // Singulo edit - Dust animation
 
```

Comment:
```suggestion
	anchored = TRUE
	density = FALSE
```
Make sure it can't be dragged around anymore (like into the SM again) and doesn't block anything.
Wasn't there a TRAIT_DUSTING check here or is that just in my stale feature branch for tg?

Reply:
Found your PR https://github.com/SinguloStation13/SinguloStation13/pull/93
Should I still apply the suggestion, or leave it for a later full port?

Reply:
I'd say apply the suggestion and after this I'll create a PR to add the trait